subsys/net/lib/sockets/CMakeLists.txt

```diff
@@ -16,5 +16,4 @@ zephyr_sources_ifdef(CONFIG_NET_SOCKETS_CAN sockets_can.c)
 endif()
 zephyr_sources_ifdef(CONFIG_NET_SOCKETS_OFFLOAD     socket_offload.c)
 
-zephyr_link_interface_ifdef(CONFIG_MBEDTLS mbedTLS)
 zephyr_link_libraries_ifdef(CONFIG_MBEDTLS mbedTLS)
```

Comment:
Here it says that the line was repeated twice.

Reply:
It's not the same line, interface vs. libraries

Reply:
d'oh, sorry, late Friday comment %)